packages/documentation/gatsby-node.js

```diff
@@ -97,93 +97,89 @@ exports.onCreateNode = ({node, getNode, actions }) => {
 
 exports.createPages = ({ graphql, actions }) => {
   const { createPage } = actions
-  return new Promise((resolve, reject) => {
-    resolve(
```

Comment:
What does removing this call do?

Reply:
The extra promise wrapping is unnecessary, so this is simplifying the code without changing any behavior. I changed it as part of my troubleshooting a build problem and left it in because it's an improvement either way.